tests/tests/swfs/avm2/bitmap_timeline/Test.as

```diff
@@ -0,0 +1,1594 @@
+﻿package {
+	public class Test {
+	}
+}
+
+function assert_divide(val1, val2) {
+	trace(val1 / val2);
+}
+
+trace("//true / true");
+assert_divide(true, true);
+
+trace("//false / true");
+assert_divide(false, true);
+
+trace("//null / true");
+assert_divide(null, true);
```

Comment:
What do any of these tests in this file have to do with Bitmap?
Is this some unfinished copy-paste artifact? :)

Reply:
Yes it is, and it's not even in the actual SWF